code/game/objects/items/cards_ids.dm

```diff
@@ -24,30 +24,48 @@
 	return BRUTELOSS
 
 /obj/item/card/data
-	name = "data disk"
-	desc = "A disk of data."
-	icon_state = "data"
+	name = "data card"
+	desc = "A plastic magstripe card for simple and speedy data storage and transfer. This one has a stripe running down the middle."
+	icon_state = "data_1"
+	obj_flags = UNIQUE_RENAME
 	var/function = "storage"
 	var/data = "null"
 	var/special = null
 	item_state = "card-id"
 	lefthand_file = 'icons/mob/inhands/equipment/idcards_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/idcards_righthand.dmi'
+	var/detail_color = COLOR_ASSEMBLY_ORANGE
 
-/obj/item/card/data/verb/label(t as text)
-	set name = "Label Disk"
-	set category = "Object"
-	set src in usr
+/obj/item/card/data/Initialize()
+	.=..()
+	update_icon()
 
-	if(usr.stat || !usr.canmove || usr.restrained())
+/obj/item/card/data/update_icon()
+	cut_overlays()
+	if(detail_color == COLOR_FLOORTILE_GRAY)
 		return
+	var/mutable_appearance/detail_overlay = mutable_appearance('icons/obj/card.dmi', "[icon_state]-color")
+	detail_overlay.color = detail_color
+	add_overlay(detail_overlay)
 
-	if (t)
-		src.name = "data disk- '[t]'"
-	else
-		src.name = "data disk"
-	src.add_fingerprint(usr)
-	return
+/obj/item/card/data/attackby(obj/item/I, mob/living/user)
+	if(istype(I, /obj/item/integrated_electronics/detailer))
+		var/obj/item/integrated_electronics/detailer/D = I
+		detail_color = D.detail_color
+		update_icon()
+
+/obj/item/proc/GetCard()
+
+/obj/item/card/data/GetCard()
+	return src
+
+/obj/item/card/data/full_color
+	desc = "A plastic magstripe card for simple and speedy data storage and transfer. This one has the entire card colored."

```

Comment:
>plastic
>make them with metal

Reply:
Hey, the IC printer somehow shits out circuit boards, LEDs, and what I can only assume are LCD screens using nothing but metal, it can probably make plastic rectangles.